.git-hooks/locale.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+
+/**
+ * What is this? With this small JavaScript file we want 
+ * to block the upload of modified files in the "locale" 
+ * folder to the repository through a git hook generated 
+ * with the Husky library.
+ * 
+ * Usage: $ node .git-hook/locale.js
+ * 
+ * Happy releasing!
+ */
+
+'use strict'
+
+// Require main dependences.
+const { execSync } = require('child_process')
+const fs = require('fs')
+const path = require('path')
+
+// Execute Git Diff for get all files with changes.
+const results = execSync('git diff --cached --name-only')
+
+// Create array with all files.
+const arrayResults = results.toString().split('\n')
+// Remove last element with empty line.
+arrayResults.pop()
+
+// Define array files to not change.
+const filesNoChange = []
+
+function gitHook(expresion) {
+
+  // Define RegExp with reg param.
+  const reg = new RegExp(expresion, 'g')
+
+  // Iterate in array for search if RegExp is true or not.
+  arrayResults.forEach(file => {
+    // Control for search if exist or not file.
+    if (fs.existsSync(path.join(__dirname, '..', file)) && reg.test(path.join(__dirname, '..', file))) {
```

Comment:
What about first creating `const filepath = path.join(__dirname, '..', file)`  and reusing it in the conditional?

Reply:
I think it's a good idea. Done!